### PR TITLE
Add cell-centered functionality to core.py

### PR DESCRIPTION
**Pull request by GarethTMorgan**
# Pull Request
### Description
Add parameter for specifying the placement of points on the grid

This pull request addresses #102

### CODE REVIEW

File: turbopy/core.py [hunk 1]
```diff
@@ -651,20 +651,27 @@ def set_value_from_keys(self, var_name, options):
...
             Number of vector components at each point.
+        placement_of_points : str, defaults to "edge-centered"
+            Designate position of points on grid
         Returns
         -------
         :class:`numpy.ndarray`
             Squeezed array of zeros.
         """
-        return np.squeeze(np.zeros((self.num_points, num_components)))
+        amended_point_number = None
```
Comment (arichar6) [inline turbopy/core.py:669]:
```suggestion
        number_of_field_points = None
```

**Comment by GarethTMorgan** [comment]
Would appreciate some eyes on the amended documentation/variable names. Not entirely sure they read well.

File: turbopy/core.py [hunk 2]
```diff
@@ -651,20 +651,27 @@ def set_value_from_keys(self, var_name, options):
...
+        if placement_of_points == "edge-centered":
+            amended_point_number = self.num_points
+        else:
```
Comment (arichar6) [inline turbopy/core.py:672]: I think I'd change this, to make "cell-centered" an explicit option. And maybe use the `else` clause to raise a `RuntimeError` (or maybe a `ValueError`) saying that an unknown placement option was specified.

File: turbopy/core.py (hunk 1 above)
Comment (arichar6) [reply turbopy/core.py:669]: I think a more specific variable name here is helpful

File: turbopy/core.py (hunk 2 above)
Comment (GarethTMorgan) [reply turbopy/core.py:672]: thank you, very good point (haha bad pun).

**Comment by arichar6** [comment]
Hmm... guess I approved, but forgot to merge this.

**Pull request merged**